frontend/api/routes.py

```diff
@@ -26,6 +26,7 @@ def define_routes(application):
     application.route("/probes", callback=probes.list)
     # files routes
     application.route("/search/files", callback=search.files)
+    application.route("/files/<filesha256>", callback=search.get_file)
```

Comment:
Cosmetic remark: I prefer to have "/files/<sha256>" just output if file is present or not (We need this too to avoid uploading an existing file). Maybe add an extra verb
/files/<sha256>/{download|get|....} for this purpose